development/xdebug/xdebug.ini

```diff
@@ -1,47 +1,1126 @@
 ; Enable xdebug extension module
 ;zend_extension="/usr/LIBDIR/php/extensions/xdebug.so"
```

Comment:
You can remove everything but these first two lines

Reply:
Done, thank you